src/Symfony/Component/Messenger/ParallelMessageBus.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger;
+
+use Amp\Parallel\Worker\ContextWorkerPool;
+use Symfony\Component\Messenger\Stamp\FutureStamp;
+
+use function Amp\async;
+use function Amp\Parallel\Worker\workerPool;
+
+class ParallelMessageBus implements MessageBusInterface
```

Comment:
```suggestion
final class ParallelMessageBus implements MessageBusInterface
```

?

Can you also add some PHPDoc to explain what's the purpose of this class?